server/sonar-server/src/main/java/org/sonar/server/property/InternalProperties.java

```diff
@@ -26,6 +26,11 @@
  * Allows to read and write internal properties.
  */
 public interface InternalProperties {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Move constants to a class or enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1214)